src/app/scheduling/daterange/date-range-form/daterange-form.component.html

```diff
@@ -0,0 +1,43 @@
+<div class="content-wrap">
+    <form class="book-flight-form" [formGroup]="twoInputForm" (ngSubmit)="submitTwoInputForm()">
+        <div class="container">
+            <div class="input-wrapper">
+                <igx-input-group class="contentA">
+                    <input #departureAirport igxInput type="text" formControlName="departureAirport">
```

Comment:
We already have a sample with scheduling, look at `Flight Booking`. If we want to create a reactive form sample for the range picker we could recreate the `Flight Booking` sample only with a reactive form.